cmd/loader/args.go

```diff
@@ -0,0 +1,100 @@
+	DatagramSize uint   `          long:"buffer-size"             default:"1500"           description:"Maximum size of datagram to send"        `
+	Workers      uint   `short:"w" long:"workers"                 default:"1"              description:"Number of parallel workers to use"       `
+	Counts       struct {
+		Counter uint64 ` short:"c" long:"counter-count"                                    description:"Number of counters to send"              `
+		Gauge   uint64 ` short:"g" long:"gauge-count"                                      description:"Number of gauges to send"                `
+		Set     uint64 ` short:"s" long:"set-count"                                        description:"Number of sets to send"                  `
+		Timer   uint64 ` short:"t" long:"timer-count"                                      description:"Number of timers to send"                `
+	} `group:"Metric count"`
+	NameCard struct {
+		Counter uint `             long:"counter-cardinality"     default:"1"              description:"Cardinality of counter names"            `
+		Gauge   uint `             long:"gauge-cardinality"       default:"1"              description:"Cardinality of gauges names"             `
+		Set     uint `             long:"set-cardinality"         default:"1"              description:"Cardinality of set names"                `
+		Timer   uint `             long:"timer-cardinality"       default:"1"              description:"Cardinality of timer names"              `
+	} `group:"Name cardinality"`
+	TagCard struct {
+		Counter []uint `           long:"counter-tag-cardinality"                          description:"Cardinality of count tags"               `
+		Gauge   []uint `           long:"gauge-tag-cardinality"                            description:"Cardinality of gauge tags"               `
+		Set     []uint `           long:"set-tag-cardinality"                              description:"Cardinality of set tags"                 `
+		Timer   []uint `           long:"timer-tag-cardinality"                            description:"Cardinality of timer tags"               `
+	} `group:"Tag cardinality"`
+	ValueRange struct {
+		Counter uint `             long:"counter-value-limit"     default:"0"              description:"Maximum value of counters minus one"     `
+		Gauge   uint `             long:"gauge-value-limit"       default:"1"              description:"Maximum value of gauges"                 `
+		Set     uint `             long:"set-value-cardinality"   default:"1"              description:"Maximum number of values to send per set"`
+		Timer   uint `             long:"timer-value-limit"       default:"1"              description:"Maximum value of timers"                 `
+	} `group:"Value range"`
+}
+
+func parseArgs(args []string) commandOptions {
+	var opts commandOptions
+	parser := flags.NewParser(&opts, flags.HelpFlag | flags.PassDoubleDash)
+	parser.LongDescription = "" + // because gofmt
```

Comment:
Why not use `` string? (I can't remember its name)

Reply:
Habit mainly.  And it doesn't look good completely left aligned in the source, but if you add spaces then go-flags strips it.  Either way, it's sub-optimal, so I went with the "embrace the suck" style.